frontend/app/controllers/Healthcheck.scala

```diff
@@ -23,7 +23,7 @@ object Healthcheck extends Controller {
     new BoolTest("Events", () => GuardianLiveEventService.events.nonEmpty),
     new BoolTest("CloudWatch", () => CloudWatchHealth.hasPushedMetricSuccessfully),
     new BoolTest("ZuoraPing", () => zuoraSoapClient.lastPingTimeWithin(2.minutes)),
-    new BoolTest("Promotions", () => promos.all.nonEmpty),
+    new BoolTest("Promotions", () => promos.futureAll.value.exists(_.map(_.nonEmpty).getOrElse(false))),
```

Comment:
It looks we had a similar issue when using promos.futureAll for the subs healthcheck and had to revert the change:
https://github.com/guardian/subscriptions-frontend/pull/758/commits/5b5af1195f62af9fe41faecdca84c79bd9c007fa